### PR TITLE
CVE-2019-14879 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14879.json [hunk 1]
```diff
@@ -61,7 +60,7 @@
         "description_data": [
             {
                 "lang": "eng",
-                "value": "moodle before versions 3.7.3, 3.6.7, 3.5.9 is vulnerable to a None."
+                "value": "A vulnerability was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier.  When a cohort role assignment was removed, the associated capabilites were not being revoked (where applicable)."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14879.json]: Please change the description to match fixed version, i.e.:

 3.7.3, 3.6.7 and 3.5.9

**Comment by cve-team** [comment]
Just waiting on implementation of iamleot's comments Dhananjay and we'll get that merged. Thanks!

**Comment by cristina479** [comment]
Red Hat CNA,

Can you address the comments in this PR in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

**Comment by darunesh-RH** [comment]
updated with new description to  match fixed version.

File: 2019/14xxx/CVE-2019-14879.json (hunk 1 above)
Comment (darunesh-RH) [reply 2019/14xxx/CVE-2019-14879.json]: Updated the description with missing information.

Comment (iamleot) [reply 2019/14xxx/CVE-2019-14879.json]: As suggested, can you please change them to match fixed version, i.e. please change:

````
[...] was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier [...]
```` 

to:

````
[...] was found in Moodle 3.7 before 3.7.3, 3.6 before 3.6.7, 3.5 before 3.5.9 and earlier [...]
````

In the latter form it's clear that a released version with the fix was released while it isn't in the current and former form.

Comment (iamleot) [inline 2019/14xxx/CVE-2019-14879.json]: Can you please change them to match fixed version, i.e. please change:

````
[...] was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier [...]
```` 

to:

````
[...] was found in Moodle 3.7 before 3.7.3, 3.6 before 3.6.7, 3.5 before 3.5.9 and earlier [...]
````

In the latter form it's clear that a released version with the fix was released while it isn't in the current and former form.

**Comment by darunesh-RH** [comment]
> Please mention fixed versions in the description as well.
> 
> Thanks!

Updated description with affected and fixed versions.

**Pull request merged**